setup.py

```diff
@@ -23,10 +23,9 @@
         "numpy",
         "imjoy-rpc>=0.2.55",
         'pathlib;python_version<"3.4"',
-        "jupyter>=1.0.0",
         "imjoy-elfinder[jupyter]",
-        "ipykernel>=5.1.4",
-        "imjoy-jupyter-extension",
+        "python-socketio[asyncio_client]==5.0.4",
```

Comment:
When we change requirements we need to sync them both here in setup.py and in the requirements files.